### DISCUSSION TITLE
QuadRotor for 4 passengers

### DISCUSSION THREAD

**Discussion by gasilva** [JSBSim-Team/jsbsim#469] [Q&A]
We will start a new model of a quadrotor for 4 passengers. 
I found some work on UAV (no passengers) with low weight.
[https://github.com/paparazzi/paparazzi/tree/master/conf/simulator/jsbsim/aircraft](https://github.com/paparazzi/paparazzi/tree/master/conf/simulator/jsbsim/aircraft)

Is there any model of a bigger quadrotor to have a start?

**Comment by seanmcleod** [comment]
I'm not aware of any large quadrotor models for JSBSim.

In terms of quadrotors and JSBSim also take a look at - https://github.com/JSBSim-Team/jsbsim/issues/333
